### PR TITLE
Use clickhouse when querying invocation stats/trend

### PR DESCRIPTION
**Pull request by luluz66**
This is protected behind a flag.

Note: code to flush invocation stats to clickhouse will be in a followup
PR.

### CODE REVIEW

File: enterprise/config/buildbuddy.local.yaml [hunk 1]
```diff
@@ -7,6 +7,8 @@ app:
   enable_target_tracking: true
 database:
   data_source: "sqlite3:///tmp/buildbuddy-enterprise.db"
+clickhouse:
```
Comment (vadimberezniker) [inline enterprise/config/buildbuddy.local.yaml:10]: I don't think we should tie the naming to a specific implementation. What if we call this 'olap_database' as an analog to 'database'?

File: enterprise/server/invocation_stat_service/invocation_stat_service.go [hunk 2]
```diff
@@ -22,12 +23,14 @@ import (
 type InvocationStatService struct {
 	env environment.Env
 	h   interfaces.DBHandle
+	ch  interfaces.ClickHouseDBHandle
```
Comment (vadimberezniker) [inline enterprise/server/invocation_stat_service/invocation_stat_service.go:26]: I'm not a big fan of these short names for struct members. `ch` is not very obvious. 
I realize there's already an existing member called `h`, but I don't think that's much better.
What we rename `ch` to `olapdb` and separately rename `h` to `db`?

File: server/interfaces/interfaces.go [hunk 3]
```diff
@@ -272,6 +272,11 @@ type DBHandle interface {
 	IsDeadlockError(err error) bool
 }
 
+type ClickHouseDBHandle interface {
```
Comment (vadimberezniker) [inline server/interfaces/interfaces.go:275]: Is our implementation strongly tied to Clickhouse or can we theoretically switch to a different column DB? Would it make sense to call this OLAPDBHandle?

**Review by vadimberezniker — CHANGES_REQUESTED** [review]

File: enterprise/server/invocation_stat_service/invocation_stat_service.go (hunk 2 above)
Comment (luluz66) [reply enterprise/server/invocation_stat_service/invocation_stat_service.go]: Done

**Review by vadimberezniker — APPROVED** [review]

**Pull request merged**